frontend/src/components/SignUp/signup.module.css

```diff
@@ -0,0 +1,131 @@
+body {
+    background-color: #F4EFFA;
+    padding-top: 10%;
+    padding-bottom: 10%;
+}
+
+h1 {
+    letter-spacing: 10px;
+    margin-bottom: 0px;
+}
+
+.create_account_subtext{
```

Comment:
I must have forgotten to mention this but generally css classes are kabob cased `create-account-subtext`. It does make accessing the CSS module style a little clunkier (e.g. `styles['create-account-subtext']`) but there are ways to fix that as well